packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -31,6 +31,7 @@ export class DdSdkReactNativeConfiguration {
     public nativeViewTracking: boolean = false;
     public proxyConfig?: ProxyConfiguration = undefined;
     public serviceName?: string = undefined;
+    public firstPartyHosts: string[] = [];
```

Comment:
Since it is a breaking change (with that one customers will stop having traces once they update the SDK), maybe we should communicate it to them. @priyanshi-gupta

Reply:
yes good point 👍